libs/product/src/drivers/magento/public_api.ts

```diff
@@ -9,10 +9,12 @@ export {
 	MagentoSimpleProduct
 } from './models/simple-product';
 
-export { DaffMagentoProductTransformerService } from './transforms/product-transformer.service';
 export { GetProductQuery } from './queries/get-product';
 export { GetAllProductsQuery } from './queries/get-all-products';
 export { magentoBundledProductFragment } from './queries/fragments/bundled-product';
 export { magentoProductFragment } from './queries/fragments/product';
 export { DaffMagentoProductService } from './product.service';
 export { DaffProductMagentoDriverModule } from './product-driver.module';
+export { transformMagentoProduct, transformManyMagentoProducts } from './transforms/product-transformers';
```

Comment:
Unless someone requests exposing this transformer, I don't think we should expose it. We should try and lower our maintenance burden as much as we can.

Reply:
ok, removed